app/views/inventories/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1> This page would be for a specific inventory </h2>
+<div class="container mt-3 d-flex flex-column align-items-center">
+  <% if @inventory.user == current_user %>
+  <h1><%= @inventory.name %></h1>
+
+  <div class="d-flex justify-content-between w-50 flex-wrap">
+    <%= link_to "Add Food", new_food_path, class: "btn btn-primary m-1" if @inventory.user == current_user %>
+  </div>
+
```

Comment:
Let's add all the foods that are in this inventory as a table view

Reply:
Hello @guerreiropedr0 , I will make this changes in the next task, thanks for the feedback